### ISSUE TITLE
Do you have any pre-processing before validation? My validation accuracy cannot reach repoted accuracy...

### ISSUE THREAD

**Issue by WeiyiLi** [d-li14/mobilenetv3.pytorch#10]
HI,
Thanks for the effort working on this. I am using the ImageNet validation dataset, which contains 50000 images, to conduct the validation for the large network. However, my accuracy on the validation set cannot reach 72%. Mine only reaches 70%. I wonder do you have any additional pre-processing for the input image during validation besides normalization? Such as cropping?
Thank you

**Comment by d-li14** [comment]
refer to #7

**Comment by tangbohu** [comment]
> HI,
> Thanks for the effort working on this. I am using the ImageNet validation dataset, which contains 50000 images, to conduct the validation for the large network. However, my accuracy on the validation set cannot reach 72%. Mine only reaches 70%. I wonder do you have any additional pre-processing for the input image during validation besides normalization? Such as cropping?
> Thank you

My run for MobileNetV3-small is also a bit worse ~2% than the reported accuarcy. Have you figure out it?
